code/modules/research/xenoarchaeology/artifact/artifact.dm

```diff
@@ -15,15 +15,10 @@
 	artifact_id = generate_artifact_id()
 
 	artifact_find_type = pick(
-	5;/obj/machinery/syndicate_beacon,
```

Comment:
This item existed since the times of olde and shall not be removed.

Reply:
the future is now old man

Reply:
old =/= good for the health of the game

I can comfortably say that the way xenoarcheology is perceived and how the job is treated as a way to just screw up the station and farm antag tokens is not a good thing for the server.

Reply:
By the love of carn don't make me leave this chair. It is the ONLY of them that isn't debug-tier gibantag and also gives the most common antag role. Not even worth the risk of using most of the time.